ignite/pkg/cosmosclient/tx.go

```diff
@@ -0,0 +1,17 @@
+package cosmosclient
+
+// TX contains transaction information.
+type TX struct {
+	// Hash contains the transaction hash.
+	Hash string
+
+	// Height of the block that contains the transaction.
+	Height int64
+
+	// BlockTime contains the timestamp of the block that contains the transaction.
+	BlockTime string
+
+	// EventLog contains the events emitted during the execution.
+	// The value is a JSON string containing the list of events.
+	EventLog string
```

Comment:
I was wondering if the normalization should be done during collection in the cosmos client. I will do it there so we use a type instead of a JSON string, it is clearer like that. The downside is that at that same point we have to serialize the whole TX to JSON, so parsing the events adds more overhead that could be moved to the caller. There is a TX field called `Raw` that is not commited yet which will contain the whole TX serialized as JSON.

We could use the tendermint result type returned by the RPC call and remove out `TX` type, that would leave the parsing responsability to the caller of the TX collector, but the issue there is that maybe we would expose more than is needed and the caller would have a dependency on tendermint.